multi-storage-file-system/csi/deploy/commands-runbook.sh

```
@@ -0,0 +1,166 @@
+
+# ============================================================
+# 1. BUILD & PUSH THE CSI DRIVER IMAGE
+# ============================================================
+
+# Prune Docker build cache if needed (frees disk for large Go builds)
+docker builder prune -af
+docker system prune -f
+
+# Build for linux/amd64 (required when building on Apple Silicon for x86 nodes)
+# Build context is multi-storage-file-system/ (not csi/)
+cd multi-storage-file-system
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)